addons/account/models/res_config_settings.py

```diff
@@ -126,6 +126,8 @@ class ResConfigSettings(models.TransientModel):
         readonly=False,
         related='company_id.account_cash_basis_base_account_id',
         domain=[('deprecated', '=', False)])
+    # TODO rename in master to account_fiscal_country_id
+    account_fiscal_country_id_copy = fields.Many2one(string="Fiscal Country Code", related="company_id.account_fiscal_country_id", readonly=False, store=False)
```

Comment:
Actually, as the definition is the same, you can call it `account_fiscal_country_id` without any issue.

Reply:
I think I tried it in the beginning and there was an error about a duplicate field in `res.config.settings`, but I could try again

Reply:
you were right, thanks! I will close upgrade PR